app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    @message = "Here is a list of all Posts by User with id: #{params[:user_id]}"
+  end
+
+  def show
+    @message = "Here is the specific Post with id: #{params[:id]} by User with id: #{params[:user_id]}"
+  end
```

Comment:
- [ ] OPTIONAL: I can see you tried to create a message here and pass it to views. I suppose you did it for testing purposes. But keep in mind that Controllers should be skinny, they should be a bridge that connects database with views. EG: 

```
  def index
    @posts = Post.all
  end

  def show
    @post = Post.find(params[:id])
  end
```

So please avoid using controllers for other purposes, you can easily create the message text in views  😉

Reply:
@ShoiraTa 
Indeed. The requirements for today is that It should show a simple message. I was experimenting with passing a value from the controller to the view combined with getting the parameter value.

Tomorrow or the day after, I suppose we will have to do what you suggests. 

The tests I included today are not to test the actual content of the page. That will be done tomorrow. Today, I'm just testing that the controllers handle the routes for Users and Posts.